OutOfSchool/OutOfSchool.WebApi/Services/ProviderService.cs

```diff
@@ -278,6 +279,17 @@ public async Task<ProviderBlockDto> Block(ProviderBlockDto providerBlockDto)
         provider.BlockReason = providerBlockDto.IsBlocked ? providerBlockDto.BlockReason : null;
         await providerRepository.UnitOfWork.CompleteAsync().ConfigureAwait(false);
 
+        // TODO What's happen if previous action will not successfull?
+        var workshops = await workshopServiceCombiner
+                           .BlockByProvider(provider)
+                           .ConfigureAwait(false);
+
+        foreach (var workshop in workshops)
+        {
+            logger.LogInformation($"IsBlocked property with povider Id = {provider?.Id} " +
+                                  $"in workshops with Id = {workshop?.Id} updated successfully.");
```

Comment:
please avoid using `null-conditional operator` everywhere.
It makes the code unreadable.